tests/unit/Utils/DocLinkTest.php

```diff
@@ -11,9 +11,11 @@
  */
 class DocLinkTest extends \PHPUnit\Framework\TestCase {
 	public function testGetServerUrl() {
-		$expected = 'https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label';
+		// This test constructs strings, from version and relative part and compares that
+		// construct with the expected result. It does not check our actual servers. 
```

Comment:
```suggestion
		// construct with the expected result. It does not check our actual servers.
```
no whitespace ant eol